internal/controller/git/git.go

```diff
@@ -336,6 +340,24 @@ func (r *repo) GetDiffPaths() ([]string, error) {
 	return paths, nil
 }
 
+func (r *repo) GetDiffPathsForCommitID(commitId string) ([]string, error) {
+	resBytes, err := libExec.Exec(r.buildCommand("show", "--name-status", "--format=format:", commitId))
```

Comment:
I was understanding the purpose of this function to be finding all paths that have had any changes since the provided commit ID. Is that correct?

If so, the command `git show --name-status --format=format:<commit id here>` does not seem to accomplish that.

I am getting the correct results from `git diff --name-only <commit id here>..HEAD`

Reply:
No, never-mind, this was getting diffs of a given commit compared to previous one, and now that I understand my approach was wrong, this does not matter, see my next comment.